StellarApp/Protocol/Planet.swift

```diff
@@ -0,0 +1,220 @@
+//
+//  Planet.swift
+//  Stellar
+//
+//  Created by TingxinLi on 4/10/19.
+//  Copyright © 2019 TingxinLi. All rights reserved.
+//
+
+import UIKit
+import ARKit
+
+enum PlanetType: String, CaseIterable{
+    
+    case sun
+    case mercury
+    case venus
+    case earth
+    case mars
+    case jupiter
+    case saturn
+    case uranus
+    case neptune
+    case pluto
+}
+
+struct RotationVector {
+    let x: CGFloat, y: CGFloat, z: CGFloat
+}
+
+protocol Rotatable {
+    func rotate()
+    func stopRotating()
+}
+
+class RotatingNode: SCNNode, Rotatable {
```

Comment:
you never made use of this class. This is a possible way to refactor but for now it's not being used so I think you should delete it